src/features/map/components/CreateMapForm.tsx

```diff
@@ -188,15 +186,14 @@ export default function CreateMapForm(): JSX.Element {
         try {
           const mapRequest: CreateMapRequest = JSON.parse(content);
           setMap(mapRequest);
-          setEditorContentError({ isPresent: false, data: undefined });
+          if (editorContentError.data === 'Invalid json.') {
+            setEditorContentError({ isPresent: false, data: undefined });
+          }
         } catch (e) {
           setEditorContentError({ isPresent: true, data: 'Invalid json.' });
         }
-        setEditorContentValidated(false);
         // eslint-disable-next-line react-hooks/exhaustive-deps
-      }, 2000),
-    [],
-  );
+      }, 2000), []);
 
   useEffect(() => {
```

Comment:
wiem ze nie to zmieniales, ale nie rozumiem czemy tutaj walidujesz szkieletora zamiast mapy? i czemu wlasciwie pusta araja?

Reply:
bo chce na początku jednorazowo wywołać validate zeby nam zwrócilo mapke do narysowania